Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -1,14 +1,53 @@
 import React from "react";
+import ReactDOM from "react-dom"
 
-const Footer = ()=>{
+class Footer extends React.Component{
+    constructor(){
+        super()
+        this.addHandler = this.addHandler.bind(this)
+        this.addHandlerHeart = this.addHandlerHeart.bind(this)
+        this.changeColorHandler = this.changeColorHandler.bind(this)
+        this.state = {
+            count : 0,
+            countHeart : 0,
+            color : ""
+        }
+    }
+
+    addHandler =()=>{
+        this.setState ((prevState)=>{
```

Comment:
Nice use of prevState, that's a concept that most people don't learn about or understand how to use until much farther along in the React journey.